checks.d/kubernetes.py

```diff
@@ -233,3 +237,27 @@ def _update_metrics(self, instance, kube_settings):
             except Exception, e:
                 self.log.error("Unable to collect metrics for container: {0} ({1}".format(
                     subcontainer.get('name'), e))
+
+        pods_list = self._retrieve_pods_list(kube_settings["pods_list_url"])
+        self._update_pods_metrics(instance, pods_list)
+
+    def _update_pods_metrics(self, instance, pods):
+        controllers_map = defaultdict(list)
+        for pod in pods['items']:
+            node_name = pod['spec']['nodeName']
```

Comment:
not sure why you use the node name, shouldn't we use the pod name instead? If it's the case you can get it from `pod['metadata']['name']`